src/DevLicenseDimo.sol

```diff
@@ -124,60 +124,65 @@ contract DevLicenseDimo is DevLicenseMeta {
 
     /**
      * @notice Issues a license in exchange for DIMO tokens.
+     * @param licenseAlias The license alias to be set.
      * @return tokenId The ID of the issued license.
      * @return clientId The ID of the client associated with the issued license.
      */
-    function issueInDimo() external returns (uint256 tokenId, address clientId) {
-        return issueInDimo(msg.sender);
+    function issueInDimo(bytes32 licenseAlias) external returns (uint256 tokenId, address clientId) {
+        return issueInDimo(msg.sender, licenseAlias);
     }
 
     /**
      * @notice Issues a new license to a specified address in exchange for DIMO tokens.
      *         Transfers spent $DIMO to the receiver address.
      * @param to The address to receive the license.
+     * @param licenseAlias The license alias to be set.
      * @return tokenId The ID of the newly issued license.
      * @return clientId The address of the license account holding the new license.
      */
-    function issueInDimo(address to) public returns (uint256 tokenId, address clientId) {
+    function issueInDimo(address to, bytes32 licenseAlias) public returns (uint256 tokenId, address clientId) {
         (uint256 amountUsdPerToken,) = _provider.getAmountUsdPerToken();
 
         uint256 tokenTransferAmount = (_licenseCostInUsd1e18 / amountUsdPerToken) * 1 ether;
```

Comment:
do we need to add a check to make sure we're never attempting to do division by zero here? (I'm wondering if an oracle call fails or something... would we ever end up accidentally using zero as `amountUsdPerToken`? I assume the tx will revert if that were to be the case...